cheddar/cheddar-integration-aws/build.gradle

```diff
@@ -2,21 +2,22 @@ apply from: '../../test.gradle'
 apply from: '../../logging-api.gradle'
 
 def springVersion = '4.1.1.RELEASE'
+def awsSdkVersion = '1.11.6'
```

Comment:
See the release notes here for version 1.11.0 of AWS SDK for Java concerning the removal of JSON.org library